resources/styles/global/progress-bars.less

```diff
@@ -7,3 +7,7 @@
   #gradient > .striped(rgba(255,255,255,.15), 135deg);
   background-size: 20px 20px;
 }
+
+.progress-bar[aria-valuenow="0"] {
```

Comment:
oh hmm, i was simplying overriding something from bootstrap, but yeah we could do that